vehicle/mkv/software/dashboard/dashboard.c

```diff
@@ -1,521 +1,159 @@
+            HV_STATE = true;
+            gpio_set_pin(HV_LED); // set HV LED
+        }
+        else {
+            HV_STATE = false;
+            READYTODRIVE = false; // Disable RTD
+            gpio_clear_pin(HV_LED); //clear HV LED
+            buzzer_counter = 0; // reset counter for next RTD cycle
+        }
+    }
+
+    //AIR Control Sense message for IMD LED
+    CANPAGE = (AIRCTRL_SENSE_MBOX << MOBNB0);
+    if (!can_poll_receive(&AIRCTRL_SENSE_FRAME)){
+        can_receive(&AIRCTRL_SENSE_FRAME, AIRCTRL_SENSE_FILTER);
+        if (AIRCTRL_SENSE_FRAME.data[6] == 0x0) { // check IMD Status
+            gpio_set_pin(IMD_LED); // set IMD light high
+        }
+        else{
+            gpio_clear_pin(IMD_LED); // set IMD light low
+        }
+    }
+
+    //Throttle message for interfacing with LED Bars Board - TODO (SPI library)
+    CANPAGE = (THROTTLE_MBOX << MOBNB0);
+    if (!can_poll_receive(&THROTTLE_FRAME)){
+        can_receive(&THROTTLE_FRAME, THROTTLE_FILTER);
+    }
 }
 
-
-/*----- Functions -----*/
-void initTimer(void) {
-	TCCR0A = _BV(WGM01);    // Set up 8-bit timer in CTC mode
-	TCCR0B = 0x05;          // clkio/1024 prescaler
-	TIMSK0 |= _BV(OCIE0A);
-	OCR0A = 0x27;           //Makes timer run at ~100Hz
-}
-
-void initIO(void) {
-	/* Initialize all inputs and outputs and interrupts */
-
-	// Enable Interrupts
-	sei();
-
-  //Sets each of these pins as Outputs
-	DDRB |= _BV(DEBUG_LED1) | _BV(DEBUG_LED2) | _BV(IMD_LED) | _BV(START_LED) | _BV(BRAKE_LED) ;
-	DDRC |= _BV(RTD_LSD) | _BV(RJ_LED1) | _BV(RJ_LED2);//| _BV(START_PIN) --> Included below
-	DDRD |= _BV(BMS_LED) | _BV(HV_LED) | _BV(LV_LED);//| _BV(STEERING_POT) --> Included below
-
-  //Set the BMS LED as OFF
-	PORT_BMS_LED &= ~_BV(BMS_LED);
-
-	//Set start pin as input
-	DDRC &= ~_BV(START_PIN);
-
-	//Set Start Pin as input pull up, Turning it on
-	PORT_START |= _BV(START_PIN);
-
-	//Set pull up resistor for steering pot
-	PORT_STEERING_POT |= _BV(STEERING_POT);
-
-	//PORTC |= _BV(PC0);
-
-	/* Setup pin change interrupt registers for start pin*/
-	PCICR |= _BV(PCIE1) | _BV(PCIE2);
-	PCMSK1 |= _BV(PCINT14);
-	PCMSK2 |= _BV(PCINT18);
-
-
-
-	/*----- Setup PWM output -----*/
-	//Output compare pin is OC1B, so we need OCR1B as our counter
-	DDRC |= _BV(PC0); //Enable output pin
-	DDRC |= _BV(PC1); //Enable output pin
-
-	// DDRD |= _BV(PD3);
-	DDRD |= _BV(PD2); //PWM is happening on this pin
-
-	// pg 119 and 110 of datasheet
-	// currently running on mode 5 for 8-bit resolution timer
-	// turns on when counter hits OCRnx and off when hits top (255)
-	TCCR1A |= _BV(COM1A1) | _BV(COM1A0) | _BV(COM1B1) | _BV(COM1B0) | _BV(WGM10);
-	TCCR1A &= ~_BV(WGM11) & ~_BV(COM1A0) & ~_BV(COM1B0);
-	TCCR1B |= _BV(CS10); //Clock prescale set to max speed
-	TCCR1B |= _BV(WGM12);
-	TCCR1B &= ~_BV(WGM13);
-
-
-	// OCR1A = (uint8_t) 115;       // Duty Cycle
-	// OCR1B = (uint8_t) 127;      // Duty Cycle
-
+//Start Button interrupt & final ReadyToDrive check
+void pcint14_callback(void) {
```

Comment:
`pcint14`? It should be something like `pcint0` or `pcint1`. What port are the pins on? PORTB? PORTC?